extensions/imessage-2fa/package.json

```diff
@@ -21,8 +22,18 @@
       "mode": "view"
     }
   ],
+  "preferences": [
+    {
+      "default": "1",
+      "description": "For searching iMessage from N days ago and now",
+      "name": "lookBackDays",
+      "required": false,
+      "title": "Search last days for 2FA",
+      "type": "textfield"
+    }
+  ],
```

Comment:
The fork action automatically changed this. No idea why.

Reply:
It just moved the position, should be fine